### ISSUE TITLE
`cor_tabular` issue with missing cells

### ISSUE THREAD

**Issue by benkeser** [CoVPN/correlates_reporting#306]
We find that function `get_rgmt` fails at the following line, if there are no cases in a particular subgroup

```r
      ret <- svyby(as.formula(sprintf("%s~%s", i, j)),
                   by=as.formula(sprintf("~%s", paste(sub.by, collapse="+"))),
                   design=design.ij,
                   svyglm, vartype="ci")
```

For example, Moderna has not adjudicated cases for baseline positive individuals and thus their `Case` status is uniformly equal to 0.

The code needs to be robust to situations where `Case` is uniformly 0 or 1 in a particular statrum.

**Comment by cyu-hvtn** [comment]
Thank you David!

**Comment by benkeser** [comment]
Nominally resolved in #317
